.travis.yml

```diff
@@ -1,5 +1,25 @@
 matrix:
   include:
+    # Job 1) Run analyzer
+    - os: linux
+      jdk: oraclejdk8
+      sudo: false
+      addons:
+        apt:
+          # Flutter depends on /usr/lib/x86_64-linux-gnu/libstdc++.so.6 version GLIBCXX_3.4.18
+          sources:
+            - ubuntu-toolchain-r-test # if we don't specify this, the libstdc++6 we get is the wrong version
+          packages:
+            - libstdc++6
+            - fonts-droid
+      before_script:
+        - git clone https://github.com/flutter/flutter.git --depth 1
```

Comment:
I'm surprised travis doesn't have helpers for the git clone step to help reduce work across their fleet.